st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -58,6 +73,37 @@ def get_all(self, **kwargs):
     def get_one(self, ref_or_id):
         return super(ActionAliasController, self)._get_one(ref_or_id)
 
+    @jsexpose(arg_types=[str], status_code=http_client.ACCEPTED)
+    def match(self, command, **kwargs):
```

Comment:
they could for the purposes of reducing the criteria of the underlying `get_all` call, which is where they are passed. So filtering by pack for example.